cg/services/run_devices/pacbio/run_file_manager/run_file_manager.py

```diff
@@ -60,11 +60,11 @@ def _get_report_files(run_path: Path) -> list[Path]:
         return report_files
 
     @staticmethod
-    def _get_hifi_read_file(run_path: Path) -> Path:
+    def _get_hifi_read_file(run_path: Path) -> list[Path]:
         """Return the path to the HiFi read file."""
         hifi_dir = Path(run_path, PacBioDirsAndFiles.HIFI_READS)
-        bam_file: Path = get_files_matching_pattern(
+        bam_files: list[Path] = get_files_matching_pattern(
             directory=hifi_dir, pattern=f"*{FileExtensions.BAM}"
-        )[0]
-        validate_files_or_directories_exist([bam_file])
-        return bam_file
+        )
+        validate_files_or_directories_exist(bam_files)
+        return bam_files
```

Comment:
Not super used to this code but I get a bit confused if this method is supposed to return a single or multiple paths. Now it changes to a list of paths and the returned variable is named `bam_files` but the method name is still `_get_hifi_read_file` in singular and the docstring also refers to "the path" instead of something in plural.

Reply:
fixed